conf/radiusd/packetfence.example

```diff
@@ -25,6 +25,7 @@ authorize {
 		&control:PacketFence-Request-Time := 0
 		&control:Load-Balance-Key := "%{Calling-Station-ID} %{User-Name}"
 	}
+	packetfence-set-realm-if-machine
```

Comment:
Do we need to do this in the inner as well since technically the username in the inner could be different

Reply:
It will cover really rare cases but we can add it.